sonar/modules/users/api.py

```diff
@@ -186,9 +186,17 @@ def delete(self, force=False, dbcommit=True, delindex=False):
         :param dbcommit: True for validating database transaction.
         :param delindex: True to remove record from index.
         """
+        from ..deposits.api import DepositRecord, DepositSearch
+
         # Remove roles from user account.
         self.remove_roles()
 
+        # Remove deposits.
+        query = DepositSearch().filter('term', user__pid=self["pid"])
```

Comment:
It would be nice if this is done asynchronously if the number of deposits is huge. This will avoid request timeout.